tests/unit/handlers/SuiChainHandler.test.ts

```
@@ -0,0 +1,759 @@
+    it('should return false when using endpoint', () => {
+      const endpointConfig = { ...mockSuiConfig, useEndpoint: true };
+      const endpointHandler = new SuiChainHandler(endpointConfig);
+
+      const supports = endpointHandler.supportsPastDepositCheck();
+
+      expect(supports).toBe(false);
+    });
+
+    it('should return false when L2 RPC is not configured', () => {
+      const noRpcConfig = { ...mockSuiConfig };
+      delete (noRpcConfig as any).l2Rpc;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)